ou_dedetai/system.py

```diff
@@ -682,7 +714,7 @@ def install_dependencies(packages, bad_packages, logos9_packages=None, app=None)
             app.root.event_generate('<<StartIndeterminateProgress>>')
         msg.status("Installing dependencies…", app)
         final_command = [
-            f"{config.SUPERUSER_COMMAND}", 'sh', '-c', "'", *command, "'"
+            f"{config.SUPERUSER_COMMAND}", 'sh', '-c', '"', *command, '"'
```

Comment:
Not sure about this, would be easier to maintain this as single quotes (to stop any weird expansions)

Reply:
I found that I needed this change to get this to work properly; there was an error I can't recall caused by the single quotes.

Reply:
This command would need to switch to double quotes if this was single https://github.com/FaithLife-Community/LogosLinuxInstaller/pull/183/files#diff-2c221b5dbbbc57e12ff8bd0cd8ca89e953d82f3e9a5356644bde9779e47ea40dR568